content/docs/details/location-management/index.md

```diff
@@ -8,7 +8,7 @@ description: How to configure locations autodiscovery
 
 The Backstage catalog uses location entities to point to external sources which define entities.
 The catalog backend will periodically retrieve information about each location and check for any updates to the entities
-defined there.
+defined there. (If using Github, these updates should be nearly immediate.)
```

Comment:
GitHub is 99% of usage, so you can typically assume that the reader is using GitHub.